tests/test_wave_03.py

```diff
@@ -128,13 +128,13 @@ def test_mark_complete_missing_task(client):
     # Assert
     assert response.status_code == 404
 
-    raise Exception("Complete test with assertion about response body")
+    # raise Exception("Complete test with assertion about response body")
```

Comment:
After commenting out this Exception, you need to write additional assertions to make the test more robust